setup.py

```diff
@@ -13,6 +15,26 @@ def convert(filename, fmt):
         with io.open(filename, encoding='utf-8') as fd:
             return fd.read()
 
+def create_mo_files():
+    data_files = []
+    localedir = 'cmsplugin_cascade/locale'
+    po_dirs = [localedir + '/' + l + '/LC_MESSAGES/'
```

Comment:
Always use ``os.path.join()`` to concatenate directories.

Reply:
Ok Thanks !
po_dirs=[ os.path.join(localedir, lang , 'LC_MESSAGES/') for lang in next(os.walk(localedir))[1]]

Reply:
Ok Thanks !
po_dirs=[ os.path.join(localedir, lang , 'LC_MESSAGES/') for lang in next(os.walk(localedir))[1]]